chain/chain/src/chain.rs

```diff
@@ -425,7 +424,7 @@ type BlockApplyChunksResult = (CryptoHash, Vec<Result<ApplyChunkResult, Error>>)
 /// Facade to the blockchain block processing and storage.
 /// Provides current view on the state according to the chain state.
 pub struct Chain {
-    store: ChainStore,
+    pub(crate) store: ChainStore,
```

Comment:
There is a store() method on Chain, can you use that instead?

Reply:
Aah, there was a `mut_store()` method that I had missed. That's why it wasn't working before.